src/webgpu/api/validation/createSampler.spec.ts

```diff
@@ -0,0 +1,19 @@
+export const description = `
+createSampler validation tests.
+
+Test Plan: 
+
+* lodMinClamp:
+  * < 0
+  * = NAN
+
+* lodMaxClamp:
+  * > MAX_SAFE_INTEGER
+  * = NAN
+  * =,> FLT_MAX
+  * < lodMinClamp
+
+* maxAnisotropy
+  * = 0
+  * = 65535 
```

Comment:
note: we don't need to add extreme value tests here, because I think the upstream spec will change to make those IDL errors (by adding EnforceRange). Filed https://github.com/gpuweb/gpuweb/issues/1187